src/sdg/templates/producten/update.html

```diff
@@ -353,7 +353,14 @@
                         {{ form.id.as_hidden }}
                         {{ form.taal.as_hidden }}
                         {% endfor %}
-                        {% localized_form formset include_form=False fields=localized_form_fields %}
+
+                        {% localized_form formset include_form=False fields=localized_form_fields  %}
+
+                        {% for form in form.forms %}
```

Comment:
Can you add a comment here (`{# ... #}` why this is needed?

I might want to refactor this completely so we don't have this "localized_form" at all but just iterate over all fields or even hardcode the included form fields each. But... later.